frontend/lib/src/components/widgets/DataFrame/columns/cells/UriOverlayEditor.tsx

```diff
@@ -64,24 +46,14 @@ const UriOverlayEditor: React.FunctionComponent<Props> = p => {
   return (
     <UriOverlayEditorStyle>
       <a
-        data-testid={"link-cell"}
+        data-testid="link-cell"
         className="gdg-link-area"
         href={uri ?? ""}
         target="_blank"
         rel="noopener noreferrer"
       >
         {preview}
       </a>
-      {!readonly && (
-        <div
-          className="gdg-edit-icon"
-          data-testid="gdg-edit-icon"
-          onClick={onEditClick}
-        >
-          <Icon content={Edit} size="lg" />
-        </div>
-      )}
-      <textarea className="gdg-input" autoFocus={true} />
```

Comment:
I wondered why this was ever needed since it was invisible to the user 🤷

Reply:
Yeah, I wondered about the same as well :) I did more extensive testing to ensure it didn't change anything but wasn't able to see anything broken.

Reply:
btw. I asked Jason about that, but he thinks that it was more of a historic artifact